eras/conway/impl/src/Cardano/Ledger/Conway/Governance.hs

```diff
@@ -541,6 +545,20 @@ data DefaultVote
     DefaultNoConfidence
   deriving (Eq, Show)
 
+instance FromCBOR DefaultVote where
+  fromCBOR = do
+    tag <- decodeWord8
+    pure $ case tag of
+      0 -> DefaultNo
+      1 -> DefaultAbstain
+      2 -> DefaultNoConfidence
+      _ -> error $ "Invalid DefaultVote tag " ++ show tag
+
+instance ToCBOR DefaultVote where
+  toCBOR DefaultNo = encodeWord8 0
+  toCBOR DefaultAbstain = encodeWord8 1
+  toCBOR DefaultNoConfidence = encodeWord8 2
+
```

Comment:
Could you also please add `EncCBOR` and `DecCBOR`, just in case. They will have default implementation that uses `To/FromCBOR`:

```suggestion
instance DecCBOR DefaultVote 

instance EncCBOR DefaultVote 
```